.evergreen/serverless/download-crypt.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+MONGODB_VERSION=${MONGODB_VERSION:-latest}
+
+DIR=$(dirname $0)
+# Functions to fetch MongoDB binaries
+. $DIR/../download-mongodb.sh
+
+get_distro
+get_mongodb_download_url_for "$DISTRO" "$MONGODB_VERSION"
+
+# Don't need to install the legacy shell
+download_and_extract_package "$MONGODB_DOWNLOAD_URL" "$EXTRACT"
+
+if [ -z $MONGO_CRYPT_SHARED_DOWNLOAD_URL ]; then
```

Comment:
can we call this code here: https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/run-orchestration.sh#L105?

Reply:
I considered that but decided against it due to the refactoring needed to have the code shared between run-orchestration.sh and create-instance.sh. I can take a look at consolidating the logic separately.

Reply:
>I can take a look at consolidating the logic separately.

that would be good